cardano-cli/test/cardano-cli-golden/Test/Golden/TxView.hs

```diff
@@ -30,60 +19,6 @@ goldenDir, inputDir :: FilePath
 goldenDir = "test/cardano-cli-golden/files/golden"
 inputDir = "test/cardano-cli-golden/files/input"
 
--- TODO: Expose command to view byron tx files
-_hprop_golden_view_byron_yaml :: Property
```

Comment:
I think this makes `HLINT` pragma unneccessary. Can you remove it if it's possible?